kickoff/jsonexportl10n.py

```diff
@@ -15,37 +15,66 @@
 from jsonexport import myjsonify
 
 
+BETA_AGGREGATION_KEYWORD = "beta"
+
+
 @app.route('/json/l10n/<releaseName>.json', methods=['GET'])
 def l10nExport(releaseName):
     # Export the l10n changeset for a product
     releaseTable = getReleaseTable(releaseName)
+
+    if releaseName.endswith(BETA_AGGREGATION_KEYWORD):
+        l10n_list = _aggregateBetaLocales(releaseTable, releaseName)
```

Comment:
we don't use "_" as coding style in ship-it, as consistency, please remove it
(I know this is bikeshedding)